lumpy/depthfilter.go

```diff
@@ -369,6 +370,33 @@ func cp(dst, src string) error {
 	return d.Close()
 }
 
+// TODO Move this if necessary
+type inter struct {
+    read1_tid int
+    read2_tid int
+    read1_pos int
+    read2_pos int
+    qname string
+}
+
+func localOutOfWindow(i, k inter) bool {
+	return (k.read1_tid > i.read1_tid || (k.read1_tid == i.read1_tid && k.read1_pos > i.read1_pos + 300))
+}
```

Comment:
That's a good point on the window size. I've only tested the 300 bp option and results looked good. The first idea I'd had was that 2 * read length should be sufficient, but perhaps it'd be more appropriate to be something like mean/median insert size ± 3 * mad/sd.

Reply:
For my test sample, the standard deviation (from the lumpy command line) was 78bp.

Reply:
yeah, it should be on the insert-size, not on the read-length. In looking through IGV at stuff that changed, I did see a true read that got removed by this as it was just over 300 bp. I think the right side can be much larger, e.g. 10KB since that's unlikely that a bad read will land within 2KB in the all vs all genome space.

Reply:
I am trying on giab with 10kb now.

Reply:
Sounds good. Alternatively, seems like we could pass in the stats from the filter struct and calculate a dynamic filter.

Reply:
here are the numbers from master, your PR, your PR with the const changed to 1000:
brentp@lorax:~/go/src/github.com/brentp/smoove$ samtools view -c hg002/hg002.disc.bam
1434810
brentp@lorax:~/go/src/github.com/brentp/smoove$ samtools view -c hg002-ernfrid/hg002.disc.bam
331256
brentp@lorax:~/go/src/github.com/brentp/smoove$ samtools view -c hg002-ernfrid10k/hg002.disc.bam
333506
```

given we get only a few extra reads with 10KB, I think that's a safer default.

Reply:
wow. that's suprising. well, I guess using the covstats.Stats.TemplateMean + 4 * covstats.Stats.TemplateSD should be somewhere in the middle.